src/main/java/com/impactupgrade/nucleus/model/AccountingTransaction.java

```diff
@@ -25,6 +25,9 @@ public class AccountingTransaction {
   public String paymentGatewayName;
   public String paymentGatewayTransactionId;
   public Boolean recurring;
+  //TODO: find a way of passing crm object custom fields to accounting layer
+  // (configurable list of custom fields' names? extract into a map to use instead of entire crmDonation object?)
+  public CrmDonation crmDonation;
```

Comment:
@VSydor Can you clarify why this is needed? Are there additional fields that we need to save in Xero itself?

Reply:
Oh nm, I see it. `updateOrCreateTransactions` only has access to the accountingTransactions, right? Do we need to refactor a bit? Have this method receive the `List<CrmContact>` directly, and then we move the translation into `List<AccountingTransaction>` here instead of upstream in XeroDataSyncService?

Reply:
Decided to add crmdonation here bz need to check custom fields in the XeroAccountingPlatformService (WPG__c, Tax_Deductible_Gift__c, Other_Income__c).

Can add List<CrmContact> directly to updateOrCreateTransactions method and then do translation inside XeroAccountingPlatformService for sure.